app/src/models/repository.ts

```diff
@@ -15,23 +15,37 @@ function getBaseName(path: string): string {
   return baseName
 }
 
+/** Base type for a directory you can run git commands successfully */
+export type WorkingTree = {
+  readonly path: string
+}
+
 /** A local repository. */
 export class Repository {
   public readonly name: string
+  /** The main working tree's path
```

Comment:
:art: as this is a multi-line comment, let's bring the first comment down to the next line to let it breathe a bit